### PR TITLE
issue: 2583763 Add CI to K8

### PR DESCRIPTION
**Pull request by igor-ivanov**
## Description
Add CI to K8

##### What
Move ci to ci-demo logic

##### Why ?
To do CI more flexible

## Change type
What kind of change does this PR introduce?
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [x] Tests
- [ ] Other

## Check list
- [ ] Code follows the style de facto guidelines of this project
- [ ] Comments have been inserted in hard to understand places
- [ ] Documentation has been updated (if necessary)
- [ ] Test has been added (if possible)

### CODE REVIEW

**Comment by igor-ivanov** [comment]
bot:retest

File: .ci/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+If proj_jjb.yaml was changed pipline configuration should be updated
+
+```
+   cd .ci
+   make jjb
+```
+
+## Job Matrix yaml
+
+The main CI steps are describe in `job_matrix.yaml` file.
```
Comment (vasily-v-ryabov) [inline .ci/README.md:38]: `are describe` -> `are described`

File: .ci/proj_jjb.yaml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        - bool:
+            name: "build_dockers"
+            default: false
+            description: "Rebuild docker containers. Check this box if .ci/DockerFile* was changed"
+        - string:
+            name: "conf_file"
+            default: ".ci/matrix_job.yaml"
+            description: "Regex to select job config file. Do not change it"
+        - string:
+            name: "DEBUG"
```
Comment (vasily-v-ryabov) [inline .ci/proj_jjb.yaml:30]: It's better to move service params to the bottom. I mean "build_dockers", "conf_file" and "DEBUG" which are all for DevOps only.

Comment (igor-ivanov) [reply .ci/proj_jjb.yaml:30]: i see differently

File: .ci/README.md (hunk 1 above)
Comment (igor-ivanov) [reply .ci/README.md:38]: done

**Review by vasily-v-ryabov — APPROVED** [review]

**Pull request merged**